daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -794,6 +794,24 @@ class RazerOrnataV3X(_MacroKeyboard):
     DEVICE_IMAGE = "https://dl.razerzone.com/src/6071-1-en-v1.png"
 
 
+class RazerOrnataV3X_02A2(_MacroKeyboard):
+    """
+    Class for the Razer Ornata V3 X 02A2
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Ornata_V3_X(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x02A2
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 1]
+    METHODS = ['get_device_type_keyboard', 'set_static_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro']
+
+    DEVICE_IMAGE = "https://dl.razerzone.com/src/6071-1-en-v1.png"
```

Comment:
make up something like https://github.com/openrazer/openrazer/blob/master/daemon/openrazer_daemon/hardware/headsets.py#L41 if the devices need the same data

Reply:
changed like the proposed solution